plugin/editorconfig.vim

```diff
@@ -544,4 +548,10 @@ endfunction "}}}1
 let &cpo = s:saved_cpo
 unlet! s:saved_cpo
 
+" {{{
+function! s:ConvertLanguage(language)
+    " Change en-GB (from the editorconfig specification) to en_gb (
+    return tolower(substitute(a:language, "-", "_", ""))
```

Comment:
I think it would be worth guarding this behind a regex match.  E.g.,
```suggestion
    if a:language =~? '\v^[a-z]{2}(-[A-Z]{2})?$'
        return tolower(substitute(a:language, "-", "_", ""))
    " ... else warn if verbose, maybe ...
    endif
```
This would make it harder for people to accidentally write `spelling_language` values that work just fine in vim but not in other editors.